glDemo/main.cpp

```diff
@@ -86,6 +100,41 @@ int main() {
 	// *** setup viewplane to the appropriate size
 	gluOrtho2D(-1.1f, 1.1f, -1.1f, 1.1f);
 
+
+	random_device rd;
+	engine = mt19937(rd());
+	range = uniform_real_distribution<float>(-1.0f, 1.0f);
+
+	random_device dr;
+	engine2 = mt19937(dr());
```

Comment:
Don't need 2nd engine - can use single engine to drive multiple ranges / distributions